src/plugins/content_management/public/components/page_render.tsx

```diff
@@ -22,22 +22,26 @@ export const PageRender = ({ page, embeddable, savedObjectsClient }: Props) => {
   const sections = useObservable(page.getSections$()) || [];
 
   return (
-    <EuiFlexGroup
-      direction="column"
-      className="contentManagement-page"
-      style={{ margin: '10px 20px' }}
-    >
-      {sections.map((section) => (
-        <EuiFlexItem key={section.id}>
-          <SectionRender
-            key={section.id}
-            embeddable={embeddable}
-            section={section}
-            savedObjectsClient={savedObjectsClient}
-            contents$={page.getContents$(section.id)}
-          />
-        </EuiFlexItem>
+    <EuiPage direction="column" className="contentManagement-page">
+      {sections.map((section, i) => (
+        <>
+          <EuiFlexItem key={section.id}>
+            <SectionRender
+              key={section.id}
+              embeddable={embeddable}
+              section={section}
+              savedObjectsClient={savedObjectsClient}
+              contents$={page.getContents$(section.id)}
+            />
+          </EuiFlexItem>
+          {i < sections.length - 1 && (
+            <>
+              <EuiSpacer size="s" />
+              <EuiSpacer size="s" />
+            </>
```

Comment:
```suggestion
         <EuiSpacer />
```
Nit: size of m is 2x of s.

Reply:
You're right, I thought the default is `m` but it's `l`